lib/mustache-parser.pegjs

```diff
@@ -4,15 +4,16 @@ start = newMustache
 // {
 //   name: string,
 //   attrs: array, (items look like 'class="abc"', 'tagName="xyz"', '(query-params foo=bar)', 'blah', '"quotedBlah"', etc)
+//   blockParams: array, (items are all strings like 'item', 'foo', etc.)
 //   modifier: '?' or '!' (optional),
 // }
 // upstream parsers will optionally add an `isEscaped:true` property
-newMustache = name:newMustacheStart m_* attrs:(m_bracketedAttrs / newMustacheAttr*) {
+newMustache = name:newMustacheStart m_* attrs:(m_bracketedAttrs / newMustacheAttr*) blockParams:m_blockParams? {
   attrs = attrs.concat(name.shorthands);
-
   var ret = {
     name: name.name,
-    attrs: attrs
+    attrs: attrs,
+    blockParams: blockParams
```

Comment:
@Bestra I'm looking here.

I have a long-term goal that we can stop parsing the `attrs` in mustaches, and just return all the content between the `{{}}` as the `name`.

But I guess that is too far out to happen now. /cc @bantic for any other thoughts.